go.mod

```diff
@@ -2,4 +2,8 @@ module github.com/aws/aws-secretsmanager-caching-go
 
 go 1.12
```

Comment:
According to the [public docs](https://aws.github.io/aws-sdk-go-v2/docs/migrating/) the Go SDK v2 requires a minimum version of Go 1.15. We should increment this value to Go 1.15 likewise.

Reply:
Agreed.